test/gui/shared/scripts/pageObjects/Activity.py

```diff
@@ -57,35 +54,41 @@ def checkFileExist(self, filename):
             }
         )
 
-    def checkBlackListedFileExist(self, filename):
+    def isResourceBlackListed(self, context, filename):
```

Comment:
confusing naming. If I would only see the function names without looking into the code I would be very confused what the difference is between `isResourceBlackListed` and `isFileBlackListed`